### PR TITLE
Fix indexing in plotfile radiation lab/COM flux

### PR DESCRIPTION
**Pull request by maxpkatz**
## PR summary

This looks like it was an error in the port from Fortran in #1700. This should address the issue in #2413.

## PR checklist

- [x] test suite needs to be run on this PR
- [ ] this PR will change answers in the test suite to more than roundoff level
- [ ] all newly-added functions have docstrings as per the coding conventions
- [ ] the `CHANGES` file has been updated, if appropriate
- [ ] if appropriate, this change is described in the docs

### CODE REVIEW

**Comment by zingale** [comment]
only 1 radiation test fails:
http://groot.astro.sunysb.edu/Castro/test-suite/gfortran/2023-04-12-002/rad-breakout.html

curiously this stores both the com and lab flux -- it is not clear to me that storing both should be allowed.

File: Source/radiation/RadPlotvar.cpp [hunk 1]
```diff
@@ -193,16 +200,16 @@ void Radiation::save_flux_in_plotvar(int level, const MultiFab& Snew,
...
 #if AMREX_SPACEDIM >= 2
             int ifiy = iflx + NGROUPS;
-            int ifoy = nflx + NGROUPS;
+            int ifoy = icomp_flux + NGROUPS;
 #endif
 
 #if AMREX_SPACEDIM == 3
             int ifiz = iflx + NGROUPS * NGROUPS;
-            int ifoz = nflx + NGROUPS * NGROUPS;
+            int ifoz = icomp_flux + NGROUPS * NGROUPS;
```
Comment (zingale) [inline Source/radiation/RadPlotvar.cpp:212]: I think that this code was wrong originally.  We dimension the flux to be of side NGROUPS * AMREX_SPACEDIM, so the offset for 3-d should be 

```
int iflz = iflx + 2 * NGROUPS;
int ifoz = icomp_flux + 2 * NGROUPS;
```

as it stands now we have a bounds error

Comment (zingale) [reply Source/radiation/RadPlotvar.cpp:212]: the dimensioning is done in `MGFLDRadSolver.cpp` line 186

Comment (maxpkatz) [reply Source/radiation/RadPlotvar.cpp]: ok, fixed

**Review by zingale — APPROVED** [review]

**Pull request merged**